tardis/montecarlo/base.py

```diff
@@ -127,11 +135,27 @@ def _initialize_geometry_arrays(self, model):
         self.r_outer_cgs = model.r_outer.to('cm').value
         self.v_inner_cgs = model.v_inner.to('cm/s').value
 
-    def _initialize_packets(self, T, no_of_packets):
+    def _initialize_packets(self, T, no_of_packets, iteration):
+        # the iteration is added each time to preserve randomness
+        # across different simulations with the same temperature,
+        # for example. We seed the random module instead of the numpy module
+        # because we call random.sample, which references a different internal
+        # state than in the numpy.random module.
+        seed = self.seed + iteration
+        if no_of_packets > MAX_SEED_VAL:
```

Comment:
this is no longer true, right?

Reply:
We still call np.random.seed in `montecarlo_main_loop`, so if an rpacket has a seed higher than `MAX_SEED_VAL`, the error will be thrown then.

This can be fixed if I change all the random calls to reflect the new NumPy random module update; I was planning to do that in a separate PR